build/common.sh

```diff
@@ -242,14 +242,10 @@ function kube::build::is_osx() {
 
 function kube::build::update_dockerfile() {
   if kube::build::is_osx; then
-    sed_opts=("-i ''")
```

Comment:
I think I figured out how to make this work: remove the double quotes here, but add them when you expand. i.e.

```shell
if kube::build::is_osx:
  sed_opts=(-i '')
else
  sed_opts=(-i)
fi
sed "${sed_opts[@]}" "s/KUBE_BUILD_IMAGE_CROSS_TAG/${KUBE_BUILD_IMAGE_CROSS_TAG}/" "${LOCAL_OUTPUT_BUILD_CONTEXT}/Dockerfile"
```